core/templates/dev/head/domain/question/EditableQuestionBackendApiService.ts

```diff
@@ -45,17 +45,18 @@ oppia.factory('EditableQuestionBackendApiService', [
       EDITABLE_QUESTION_DATA_URL_TEMPLATE, QUESTION_CREATION_URL,
       QUESTION_SKILL_LINK_URL_TEMPLATE) {
     var _createQuestion = function(
-        skillId, questionDict, successCallback, errorCallback) {
+        skillIds, questionDict, successCallback, errorCallback) {
       var questionCreationUrl = UrlInterpolationService.interpolateUrl(
         QUESTION_CREATION_URL, {
-          skill_id: skillId
+          skill_ids: skillIds.join(',')
```

Comment:
Make sure to rename these fields correctly

Reply:
Done.